session/session_test.go

```diff
@@ -151,7 +151,7 @@ func (s *testSessionSuite) TestErrorRollback(c *C) {
 	var wg sync.WaitGroup
 	cnt := 4
 	wg.Add(cnt)
-	num := 100
+	num := 20
```

Comment:
Is it OK to decrease the tested number?

Would you please give some explains why this test become slower since the related PR is merged

Reply:
> Is it OK to decrease the tested number?

I guess so, at lease the change doesn't affect the correctness.

The `ResolveLocks` function is more sensitivity than before, in a heavy write conflict sense the contention increase.